01-node-tutorial/answers/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules  
```

Comment:
Best practices is usually only one gitignore per repo - not blocking and doesn’t break anything